.github/workflows/validate_release.yml

```diff
@@ -0,0 +1,62 @@
+name: Validate release
+
+on:
+  workflow_call:
+    inputs:
+      path:
+        description: 'Go mod directory (e.g. "plugins/source/aws", "plugins/destination/csv", "cli")'
+        type:        string
+        required:    true
+      runner:
+        description: 'Runner to be used (default: "ubuntu-latest")'
+        type:        string
+        required:    false
+        default:     'ubuntu-latest'
+jobs:
+  validate-release:
+    # this if check is here as we check for 'validate-release / validate-release' run
+    if:              startsWith(github.head_ref, 'release-please--branches--main--components') || github.event_name == 'push'
+    timeout-minutes: 60
+    runs-on:         ${{ inputs.runner }}
+    env:
+      CGO_ENABLED: 0
+    steps:
+    - id:  dash-path
+      run: |
+           echo "dashed=$(echo ${{ inputs.path }} | tr '/' '-')" >> $GITHUB_OUTPUT
```

Comment:
```suggestion
```

I think we can drop this dashed logic and just used the value as is WDYT? It's used only for the fallback release cache key so we could simplify this logic

Reply:
I don't think the cache key can include `/` symbol (that's why the dashed logic is here).
Additionally, this step allows us to have essentially the same `restore-keys` that are now hardcoded (in plugin name part sense)

Reply:
See my comment here https://github.com/cloudquery/cloudquery/pull/9780#discussion_r1161604359

Reply:
removed in 07016b2